### ISSUE TITLE
Run exported comp through FS.

### ISSUE THREAD

**Issue by philderbeast** [FAI-CIVL/FAI-Airscore#120]
A good test of `*.fsdb` export would be to open it in FS, rescore and then save. The file won't be the same because FS adds timestamps for when scoring happened. The scores might be different, the outputs. The inputs though should be the same.

Perhaps this could be done as a one off check today. If we could run FS headless and if we could extract the input subset from the `*.fsdb` that `fs-filter` can do, then we could add this as an automated test one day.

**Comment by biuti** [comment]
Output would be different in PWC2019/GAP2020 as we calculate the LC starting from start_time, and not time first pilot crossed start gate.
With LC weighting that should not make a huge difference.
I can try tho.
Did you try using Airscore FSDB file in fs-filter?

**Comment by philderbeast** [comment]
> Did you try using Airscore FSDB file in fs-filter?

Yes, @biuti not only `fs-filter` but all of the commands for batch scoring the comp I've been given, exported by airscore. I have added a [label](https://github.com/BlockScope/flare-timing/issues?q=is%3Aissue+label%3Apg-italy-2020) for that comp. There's something odd parsing out the *norm* scores with `fs-score`. Once I get that sorted, I'll publish the comp:

```
Italian Open 2020
2020-08-01 to 2020-08-08, Santa Elisabetta
```
